nss-day-cohort-41/tabloidmvc-the-gefilte-fish
# Dl editcategory

# Description
Implemented ability to edit a category name from the Category Management section.
Fixes #12 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. Make sure all seed data is up-to-date
2. Start the program through the debugger or ```dotnet run```
3. Login as ```admin@example.com```
4. Navigate to Admin Tools > Category Manager
5. Try editing an existing category, or create a category and try to edit it.
6. Navigating to //```<your localhost>```/category/edit/14 should redirect to an error page
# Checklist:
